dnf/base.py

```diff
@@ -1993,7 +1993,10 @@ def provides(self, provides_spec):
         providers = dnf.query._by_provides(self.sack, provides_spec)
         if providers:
             return providers, [provides_spec]
-        binary_provides = [prefix + provides_spec for prefix in ['/usr/bin/', '/usr/sbin/']]
+        if provides_spec.startswith('/bin/') or provides_spec.startswith('/sbin/'):
+            binary_provides = ['/usr' + provides_spec]
```

Comment:
does it mean that if I do `dnf provides /sbin/foo`, then it would only look into `/usr/sbin/foo`? I think this would break for some of packages which still didn't do UsrMove.

Reply:
Previous behavior was even worse - dnf looked for "/usr/sbin//sbin/foo". Package which didn't do UsrMove should be found by the first `.filterm(file__glob=provides_spec)` without manipulating with provides_spec.

Reply:
You are true, it can be broken when you do `dnf provides foobinary` and this foobinary does not live in /usr/(s)bin. I'll fix it.